package/yast2-storage-ng.spec

```diff
@@ -26,12 +26,12 @@ Source:		%{name}-%{version}.tar.bz2
 # Yast2::FsSnapshots.configure_on_install=
 Requires:	yast2 >= 4.0.24
 Requires:	yast2-ruby-bindings
-# BlkDevice#usable_as_blk_device? and Partitionable#usable_as_partitionable?
-Requires:	libstorage-ng-ruby >= 3.3.160
+# New partition table default is now GPT, no longer MSDOS
+Requires:	libstorage-ng-ruby >= 3.3.169
 
 BuildRequires:	update-desktop-files
-# BlkDevice#usable_as_blk_device? and Partitionable#usable_as_partitionable?
-BuildRequires:	libstorage-ng-ruby >= 3.3.160
+# New partition table default is now GPT, no longer MSDOS
+BuildRequires:	libstorage-ng-ruby >= 3.3.169
```

Comment:
As commented on the trello card, synchronizing the value of both defaults (libstorage-ng and Y2Storage) may be a nice to have bonus. But I don't think it's the real fix. The real fix would be to use the Y2Storage default in the partitioner, so we don't rely on it being in sync with the library default (no other part of YaST relies on that).

Reply:
Yes, the partitioner does use the default - whatever comes from the library.

I changed those version numbers only for the unit tests, but your solution below is the better approach, so I can revert that new requirement to avoid too close coupling of those versions without really being forced to it.